crates/lsp/src/lib.rs

```diff
@@ -134,13 +101,9 @@ impl LspService for NargoLspService {
 // and params passed in.
 
 fn on_initialize(
-    state: &mut LspState,
-    params: InitializeParams,
+    _state: &mut LspState,
+    _params: InitializeParams,
 ) -> impl Future<Output = Result<InitializeResult, ResponseError>> {
-    if let Some(root_uri) = params.root_uri {
-        state.root_path = root_uri.to_file_path().ok();
-    }
```

Comment:
We could use the root path to build the entire project (even workspace manifest) upon initialize, but I'm not sure if we actually want to do that